src/OrchardCore.Modules/OrchardCore.Queries/Views/Items/QueryBasedContentDeploymentStep.Fields.Edit.cshtml

```diff
@@ -0,0 +1,46 @@
+@using OrchardCore.Queries.ViewModels
+@model QueryBasedContentDeploymentStepViewModel
+@inject OrchardCore.ContentManagement.Metadata.IContentDefinitionManager ContentDefinitionManager
+@inject OrchardCore.Queries.IQueryManager QueryManager
+
+<style asp-name="codemirror"></style>
+<script asp-name="codemirror" depends-on="admin" at="Foot"></script>
+<script asp-name="codemirror-mode-javascript" at="Foot"></script>
+
+<h5>@T["Queried Content Items"]</h5>
+<span class="hint">@T["The name of the Query which will be used to fetch content items."]</span>
+
+<div class="mb-3">
+    <div class="form-check">
+        <input type="checkbox" class="form-check-input" asp-for="ExportAsSetupRecipe" />
+        <label class="form-check-label" asp-for="ExportAsSetupRecipe">@T["Export As Setup recipe"]</label>
+        <span class="hint">@T["Check if the data should be exported as a Setup recipe."]</span>
+    </div>
+    <div class="alert alert-warning">@T["If checked, you will have to manually declare variables for content item references like Content Picker Fields, Taxonomy Fields, Tags Fields, and List item IDs and replace them where needed in the recipe. Any reference to these content items will be lost."]</div>
+</div>
+
+<div class="mb-3" asp-validation-class-for="QueryName">
+    <label asp-for="QueryName">@T["Query"]</label>
+    <select class="form-select form-select-sm" asp-for="QueryName">
+        @foreach (var query in (await QueryManager.ListQueriesAsync()))
+        {
+            var name = query.Name;
+            <option value="@name">@name</option>
+        }
+    </select>
+    <span asp-validation-for="QueryName" class="text-danger"></span>
+    <div class="hint">@T["The name of the Query which will be used to fetch content items. NOTE: The Query must be configured to return Documents."]</div>
```

Comment:
Can we make the note as info alert that show up above as we normally did

Reply:
not sure what you are requesting as we have an alert:

![image](https://user-images.githubusercontent.com/6403130/180607516-005040a4-2ef2-49ca-b4db-1fb1ea199f51.png)

Reply:
Simply add an info alert similar to what you did in Export As Setup recipe, or simply surround the note with strong tag. My intend to make the note catch the eyes, coz the hint is grayed

Hope it's clear

Reply:
I think you are mixing two different things **field hint** and **field validation**.
Everywhere in Orchard Core, field hints are grayed:
![image](https://user-images.githubusercontent.com/6403130/180608841-18c849c3-e26e-452f-ab59-8b25ffa6f9f4.png)

Reply:
If you looked into the pic in https://github.com/OrchardCMS/OrchardCore/pull/12045#discussion_r928124991, the note is grayed like other text, probably no one can recognize there's a very importnant note about return result, so my intend to make it bold, then anyone can recognize it

Reply:
![image](https://user-images.githubusercontent.com/6403130/180611784-a480b3e2-5ee2-4348-9bd4-e006f3fdd776.png)

Reply:
No need to captalize NOTE,  you can use Note with surrounded strong or b tag. One more thing can we move it above where the notification showed up, or let's hear @Piedone feedback

Reply:
While I suggested it, now that we have validation, yeah, perhaps having "NOTE" as all caps is unnecessary.